### PR TITLE
test code with mock tikv

### PR DESCRIPTION
**Pull request by hanfei1991**
Use mocktikv to write test, but test is yet to complete. More PRs need to add.

### CODE REVIEW

File: src/test/mock_tikv.h [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        auto & ostream = sess.sendRequest(req);
+        ostream << "true";
+    }
+    void disableFailPoint(int store_idx, std::string fail_point)
+    {
+        const Store & store = stores[store_idx];
+        HTTPClientSession sess("127.0.0.1", 2378);
+        HTTPRequest req(HTTPRequest::HTTP_DELETE,
+            std::string(mock_server) + "/mock-tikv/api/v1/clusters/" + std::to_string(id) + "/stores/" + std::to_string(store.id)
+                + "/failpoints/" + fail_point);
```
Comment (disksing) [inline src/test/mock_tikv.h]: req is not sent

File: src/test/server_is_busy_get_test.cc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ClusterPtr global_cluster;
+
+bool testReadIndex()
+{
+    global_cluster = initCluster();
+    std::vector<std::string> pd_addrs = global_cluster->pd_addrs;
+
+    pd::ClientPtr clt = std::make_shared<pd::Client>(pd_addrs);
+
+    global_cluster->enableFailPoint(0, "server-busy");
```
Comment (disksing) [inline src/test/server_is_busy_get_test.cc]: failpoint not disabled

Comment (hanfei1991) [reply src/test/server_is_busy_get_test.cc]: disable on next pr

File: src/test/mock_tikv.h (hunk 1 above)
Comment (hanfei1991) [reply src/test/mock_tikv.h]: test disable on next pr

**Pull request merged**